src/net.cpp

```diff
@@ -2171,7 +2174,7 @@ void CNode::AbortMessage() UNLOCK_FUNCTION(cs_vSend)
     LogPrint("net", "(aborted)\n");
 }
 
-void CNode::EndMessage() UNLOCK_FUNCTION(cs_vSend)
+void CNode::EndMessage(const char* pszCommand) UNLOCK_FUNCTION(cs_vSend)
```

Comment:
`CNode::BeginMessage()` also uses `const char*` (https://github.com/bitcoin/bitcoin/blob/master/src/net.h#L499). `EndMessage()` is always called over `PushMessage(const char* pszCommand, const T1& a1, ...)`. Not sure if it would make sense to cast/transform to a `std::string`.